src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,127 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int MINIMUM_LENGTH_ARRAY = 10;
+
+    private static final double CAPACITY_MULTIPLIER = 1.5;
+
+    private T[] values = (T[]) new Object[MINIMUM_LENGTH_ARRAY];
```

Comment:
fix according to common mistakes

Reply:
I don't understand what I must change

Reply:
move initialization for this array to consctructor